candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```diff
@@ -22,6 +23,7 @@ apiVersions:
           type: string
       masterNodeGroup:
         type: object
+        default: {}
```

Comment:
Why we need this default ?

Reply:
These defaults are not needed, I forgot to remove them after the tests.